tests/lib/reset.sh

```diff
@@ -99,7 +99,7 @@ reset_classic() {
                 EXTRA_NC_ARGS=""
                 ;;
         esac
-        while ! printf 'GET / HTTP/1.0\r\n\r\n' | nc -U $EXTRA_NC_ARGS /run/snapd.socket; do sleep 0.5; done
+        while ! printf 'GET / HTTP/1.0\r\n\r\n' | nc -U "$EXTRA_NC_ARGS" /run/snapd.socket; do sleep 0.5; done
```

Comment:
I think this may be wrong actually. With quotes you'll get `.. | nc -U "-q 1" /run/snapd.socket`, but we want the non-quotes version, `... | nc -U -q 1 /run/snapd.socket`.

Reply:
oh hmm shellcheck complained about this, perhaps we just need a shellcheck ignore thing, but also I was using newer shellcheck than we have in spread so perhaps that is why it didn't get flagged before ?

Reply:
reverted this change